src/pages/game/is-guessed-toast/isGuessedToast.tsx

```diff
@@ -0,0 +1,28 @@
+import { Alert, Snackbar } from '@mui/material';
+
+import { useAppSelector } from '../../../app/hooks';
+import { selectGuessedWords } from './isGuessedToastSlice';
+
+type PropsType = {
+    autoHideDuration: number;
+    onToastClose: () => void;
+};
+
+const AlreadyGuessedToast = ({ autoHideDuration, onToastClose }: PropsType) => {
+    const { isAlreadyGuessed } = useAppSelector(selectGuessedWords);
+
+    return (
+        <Snackbar
+            open={isAlreadyGuessed}
+            autoHideDuration={autoHideDuration}
+            onClose={onToastClose}
+            anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+        >
+            <Alert severity="warning" onClose={onToastClose}>
```

Comment:
Please change styling of this to be in line with our styles. Its not real warning but a part of the game

Reply:
Fixed.